contracts/wevmos.go

```diff
@@ -0,0 +1,30 @@
+// Copyright Tharsis Labs Ltd.(Evmos)
+// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/evmos/blob/main/LICENSE)
+
+package contracts
+
+import (
+	_ "embed" // embed compiled smart contract
+	"encoding/json"
+
+	evmtypes "github.com/evmos/evmos/v18/x/evm/types"
+)
+
+var (
+	//go:embed compiled_contracts/WEVMOS.json
+	WevmosJSON []byte
```

Comment:
nit: to be consistent also with the naming used in the file
```suggestion
	WEVMOSJSON []byte
```